bucket/eiskaltdcpp.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "2.4.2",
+    "description": "File sharing program using DC and ADC protocols",
+    "homepage": "https://github.com/eiskaltdcpp/eiskaltdcpp",
+    "license": "GPL-3",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/eiskaltdcpp/files/Windows/EiskaltDC++-2.4.2_x86_64-portable.7z",
+            "hash": "sha1:9ed0cd684989e40a29d58e85960f908d780774c0"
+        },
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/project/eiskaltdcpp/files/Windows/EiskaltDC++-2.4.2_x86-portable.7z",
+            "hash": "sha1:764df76901b038d460909f4422950082c70c6869"
+        }
+    },
+    "extract_dir": "EiskaltDC++-2.4.2_x86_64",
+    "bin": "eiskaltdcpp-daemon.exe",
+    "shortcuts": [
+        [
+            "EiskaltDC++.exe",
+            "EiskaltDC++"
+        ]
+    ],
+    "checkver": {
+        "url": "https://sourceforge.net/projects/eiskaltdcpp/rss?path=/Windows",
+        "regex": "portable"
+    },
+    "autoupdate": {
```

```suggestion
    "persist": [
        "Certificates",
        "FileLists",
        "HubLists",
        "FinishedDownloads.sqlite",
        "FinishedUploads.sqlite",
        "HashData.dat"
    ],
    "autoupdate": {
```

This will keep those files untouched, so that we can **keep the settings untouched** when users update the app via *Scoop*.